### PR TITLE
Fixes #16217: relayd admin guide

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/16217

### CODE REVIEW

File: src/reference/modules/administration/pages/relayd.adoc [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+Relay also reads the following data files:
+
+* `/var/rudder/lib/relay/nodeslist.json`: contains information about the nodes behind a relay
+* `/var/rudder/lib/ssl/allnodescerts.pem` (only on root server): contains the certificates of all nodes managed by the root server (only on root server)
+
+Configuration reload uses the reload API endpoint, which allows `systemctl reload rudder-relayd`
+to be synchronous (and ensure configuration is already applied when command returns).
+
+Reloading relayd configuration reloads all data files and the `logging.conf` file,
```
Comment (fanf) [inline src/reference/modules/administration/pages/relayd.adoc]: I needed several pass to understand that sentence, I wasn't understanding the opposition introduced by the comma. Perhaps:
```
Reloading relayd configuration reloads all data files and update properties from
`logging.conf` but not the one from `main.conf`. Changing properties in `main.conf`
requires a restart to be taken into account. 
```

File: src/reference/modules/administration/pages/relayd.adoc [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+reloading `main.conf` requires a restart.
+
+== Data files
+
+Relayd operates on different data directories:
+
+* `/var/rudder/inventories` (on on simple relays): to forward inventories to upstream relay or root server. On root server, the webapp monitors this folder directly.
+* `/var/rudder/reports`: to forward reports to upstream relay, or, on root server, to parse and insert reports into te database.
```
Comment (fanf) [inline src/reference/modules/administration/pages/relayd.adoc]: s/te/the

**Comment by amousset** [comment]
PR updated with a new commit

File: src/reference/modules/administration/pages/relayd.adoc [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+Relay also reads the following data files:
+
+* `/var/rudder/lib/relay/nodeslist.json`: contains information about the nodes behind a relay
+* `/var/rudder/lib/ssl/allnodescerts.pem` (only on root server): contains the certificates of all nodes managed by the root server
+
+Configuration reload uses the reload API endpoint, which allows `systemctl reload rudder-relayd`
+to be synchronous (and ensure configuration is already applied when command returns).
+
+Reloading relayd configuration reloads all data files and update properties from
+`logging.conf` but not the one from `main.conf`. Changing properties in `main.conf`
```
Comment (fanf) [inline src/reference/modules/administration/pages/relayd.adoc]: the ones

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**